Trackers/iou_pred_tracker/iou_pred.py

```diff
@@ -0,0 +1,222 @@
+import numpy as np
+from ..track import Track
+from scipy.optimize import linear_sum_assignment as linear_assignment
+import math
+
+class iou_pred_track(Track):
+    def __init__(self, id, bbox, hits, miss):
+        self.history = np.array([[0,0,0,0],[0,0,0,0],[0,0,0,0]])
```

Comment:
In my opinion history should be maintained as a list and can be converted to np.array whenever required, or maybe any other better approach rather then direct providing a hardcoded initialization.